spec/models/admin_user_spec.rb

```diff
@@ -0,0 +1,22 @@
+describe AdminUser do
+  it 'return type should be User' do
```

Comment:
Tests for a method are grouped together in a separate `describe` block (see our other test files). In this case it would be `describe '.find_or_create'`.